checkov/json_doc/runner.py

```diff
@@ -1,19 +1,35 @@
 from __future__ import annotations
 
-from typing import Any
+from typing import Any, TYPE_CHECKING
 
-from checkov.common.checks.base_check_registry import BaseCheckRegistry
 from checkov.common.bridgecrew.check_type import CheckType
 from checkov.common.parsers.json import parse
 from checkov.common.parsers.node import DictNode
 from checkov.common.runners.object_runner import Runner as ObjectRunner
 
+if TYPE_CHECKING:
+    from checkov.common.checks.base_check_registry import BaseCheckRegistry
+    from checkov.common.graph.db_connectors.networkx.networkx_db_connector import NetworkxConnector
+    from checkov.common.runners.graph_builder.local_graph import ObjectLocalGraph
+    from checkov.common.runners.graph_manager import ObjectGraphManager
+
 
 class Runner(ObjectRunner):
     check_type = CheckType.JSON  # noqa: CCE003  # a static attribute
 
-    def __init__(self) -> None:
-        super().__init__()
+    def __init__(
+        self,
+        db_connector: NetworkxConnector | None = None,
+        source: str | None = None,
```

Comment:
@gruebel Shouldn't this be by default `json`?

Reply:
good question, yeah lets add it as a default.